blog/series-b-press.md

```diff
@@ -0,0 +1,59 @@
+  - **Netflix**: chose Temporal as the foundation for its next gen CI/CD system which will be used by almost every Netflix developer. “We’ve been using Temporal as a critical and core component in developing our next-gen CI/CD platform at Netflix, where it’s being used to help orchestrate fast, safe, and reliable software delivery to multiple cloud providers’ data centers around the world,” said Rob Zienert, Netflix senior software engineer. “Netflix engineers spend less time writing logic to maintain application consistency or guard against failures because Temporal does it for them. The Temporal platform is easy to operate and fits naturally in our development workflow.”
+  - **Descript**: relies on Temporal to eliminate production incidents. "Descript had one incident every week just on the transcription workflow because it was too complicated to maintain,” said Nicolas Gere-lamaysouette, software engineer at Descript. “Engineers can test each worker separately, but it was impossible to test the logic for every new feature end-to-end, so they were afraid of doing any changes in that code path. The frequency of production incidents has declined from once-a-week to virtually zero." 
+  - **Instacart**: seamlessly migrated multiple core systems to Temporal and now runs over 75 million monthly workflows. “The Temporal technology has been core to many new Instacart offerings, growing rapidly and without drama to over 75 million monthly workflows across vastly different use cases within only a few months after production launch,” said Josh Wickham, Instacart engineering manager, systems and frameworks. “Temporal the company and community have been key to that success, providing Instacart with clear, timely assistance and guidance for how to use the product at such a large scale.”
+  - **Datadog**: leverages Temporal for database reliability and developer experience, boasting over 100 internal Temporal users. 
+  - **Snap**: is a vocal early adopter of Temporal, embracing it as a cross-organization solution for services orchestration and infrastructure deployments. “Temporal Cloud provides Snap a highly reliable and scalable foundation for Snap Stories, enabling us to deliver the amazing global experience our users have come to expect,” said Saral Jain, Director Of Engineering, Enterprise and Cloud Services at Snap Inc.
+
+
+## About Temporal
+
+The Temporal platform enables developers to create reliable applications using familiar tools that make them productive. Temporal is used by Netflix, Doordash, Snap, Box, Stripe, Hashicorp, Coinbase and others. Developers are empowered to build and operate resilient applications using developer-friendly primitives. Developers write business logic - not glue code - and Temporal takes care of the rest. More information is available at temporal.io.
```

Comment:
Em dashes added per last para of https://developers.google.com/style/dashes#em-dashes.

```suggestion
The Temporal platform enables developers to create reliable applications using familiar tools that make them productive. Temporal is used by Netflix, Doordash, Snap, Box, Stripe, Hashicorp, Coinbase, and others. Developers are empowered to build and operate resilient applications using developer-friendly primitives. Developers write business logic—not glue code—and Temporal takes care of the rest. More information is available at temporal.io.
```